suite-native/accounts/src/components/TokenListItem.tsx

```diff
@@ -33,20 +34,28 @@ const horizontalLine = prepareNativeStyle(utils => ({
     height: utils.spacings.medium,
     borderLeftColor: utils.colors.borderDashed,
     borderLeftWidth: 1,
-    marginLeft: utils.spacings.medium + utils.spacings[TOKEN_ICON_SIZE] / 2,
+    marginLeft: utils.spacings.medium + utils.spacings.large / 2,
 }));
 
-export const TokenListItem = ({ symbol, balance, isLast, label }: TokenListItemProps) => {
+export const TokenListItem = ({
+    symbol,
+    balance,
+    isLast,
+    label,
+    accountKey,
+    onSelectAccount,
+}: TokenListItemProps) => {
     const { applyStyle } = useNativeStyles();
-    const [isBottomSheetOpen, setIsBottomSheetOpen] = useState(false);
 
-    const handleOpenBottomSheet = () => {
-        setIsBottomSheetOpen(true);
+    const handleOnPress = () => {
+        onSelectAccount(accountKey, symbol);
     };
 
+    const iconName = (symbol in ethereumTokenIcons ? symbol : 'erc20') as EthereumTokenIconName;
```

Comment:
nit: you can move this outside of the component

Reply:
`erc20` is part of `ethereumTokenIcons` - are you sure this will work?

Reply:
Yes, it works. 'erc20' is in the `etherumTokenIcons`, but `symbol` will be never equal to `erc20`. Even if it would be, it would work.